docs/docs/cmd/entra/multitenant/multitenant-add.mdx

```diff
@@ -0,0 +1,108 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# entra multitenant add
+
+Creates a new multitenant organization
+
+## Usage
+
+```sh
+m365 entra multitenant add [options]
+```
+
+## Options
+
+```md definition-list
+`-n, --displayname <displayName>`
+: Display name for the multitenant organization.
+
+`-d, --description [description]`
+: Description for the multitenant organiztion.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be at least **Security Administrator**.
+
+:::
+
+## Examples
+
+Create a new multitenant organization with display name only
+
+```sh
+m365 entra multitenant add --displayName 'Contoso organization'
+```
+
+Create a new multitenant organization with display name and description
+
+```sh
+m365 entra multitenant add --displayName 'Contoso organization' --description 'Multitenant organization between Contoso, Fabrikam, and Woodgrove Bank'
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+    "id": "c231d4a5-138f-4bdf-9fee-f26a90ea3ad0",
+    "createdDateTime": "2024-05-05T05:05:05Z",
+    "state": "active",
+    "displayName": "Contoso organization",
+    "description": "Multitenant organization between Contoso, Fabrikam, and Woodgrove Bank"
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  createdDateTime: 2024-05-05T05:05:05Z
+  description    : Multitenant organization between Contoso, Fabrikam, and Woodgrove Bank
+  displayName    : Contoso organization
+  id             : c231d4a5-138f-4bdf-9fee-f26a90ea3ad0
+  state          : active
+
+  ```
+
```

Comment:
````suggestion
  ```text
  createdDateTime: 2024-05-05T05:05:05Z
  description    : Multitenant organization between Contoso, Fabrikam, and Woodgrove Bank
  displayName    : Contoso organization
  id             : c231d4a5-138f-4bdf-9fee-f26a90ea3ad0
  state          : active
  ```

````